### PR TITLE
247 Update pipeline documentation

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
This PR: 
- resolves https://github.com/ynput/ayon-documentation/issues/247
- update the structure of AYON Server section a little bit which should make it clearer.
image

## Additional info
We may not need to explain the difference between `update pipeline` and `server bootstrapping` because `server bootstrapping` no longer exists as far as I know.

## Testing notes:
1. Read through the changes.
2. Approve if they make sense to you.

### CODE REVIEW

File: website/docs/admin_server_updating_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+id: admin_server_updating_pipeline
+title: Updating Server Pipeline to latest Release
+sidebar_label: Updating Server Pipeline
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md:4]: Your feedback makes sense.

**Review by Innders — CHANGES_REQUESTED** [review]
Nice one! I have made some very minor changes for you to approve.

Could you please use this image instead of the currently vertical one. It looks slightly nicer and will flow with the page better as it's landscape.

update_pipeline_release

Could we please also mention that a new bundle will be created for the new release, but it's up to you to test it and put it in production.

Do we have documentation on migrating settings between addons? This tools is best used alongside the copy settings between bundles tool and it would be great to reference that here.

The most likely workflow is:

1. Use Update Pipeline tool.
2. Use [Copy bundle settings tool](https://community.ynput.io/t/ayon-server-1-3-9-settings-migration-tool/1734#copy-addon-settings-between-bundles-2) to copy settings to new release bundle.
3. Test new bundle and set to production.

To do list:

- [ ] Approve suggestions
- [x] Use new image
- [x] Document new bundle is created after finishing update
- [x] Mention that the next steps would be to use the Copy Bundle Settings Tool, test and set to production. (In the future we can link to more details docs)

File: website/docs/admin_server_updating_pipeline.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md]: I've updated the name in https://github.com/ynput/ayon-documentation/pull/266/commits/d83eb5314ccd586722e0292cb057bb8a4e206715

File: website/docs/admin_server_updating_pipeline.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<div class="col col--6 markdown">
+
+![](assets/server/admin/update_pipeline.png)
+
+</div>
+</div>
+
+:::info
+Once the update starts, you can't stop it, but it shouldn't take long.
+:::
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md]: Thank you, I've added it in https://github.com/ynput/ayon-documentation/pull/266/commits/79a3242bc90a9fa66bfb55ba96500a69f91b647a.

File: website/docs/admin_server_updating_pipeline.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+:::tip Pipeline releases
+Pipeline releases are official bundles with the latest tested and stable addons, launchers and their dependencies for your pipeline.
+:::
+
+<div class="row markdown">
+<div class="col col--6 markdown">
+
+The `Update Pipeline` option is designed to be user-friendly, transparent, and fast. it automatically detects which addons need updates and identifies the platforms you’re already using (whether it’s MacOS, Windows, or Linux). The update process only requires two simple clicks, select the menu option, and hit the install button.
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md]: Thank you, I've added it in https://github.com/ynput/ayon-documentation/pull/266/commits/79a3242bc90a9fa66bfb55ba96500a69f91b647a.

**Comment by MustafaJafar** [comment]
@Innders Thanks a ton for your suggestions. I've pushed more updates.

File: website/docs/admin_server_updating_pipeline.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+:::tip Pipeline releases
+Pipeline releases are official bundles with the latest tested and stable addons, launchers and their dependencies for your pipeline.
+:::
+
+The `Update Pipeline` tool is designed to be user-friendly, transparent, and fast. It automatically detects which addons need updates and identifies the platforms you’re already using (whether it’s MacOS, Windows, or Linux). The update process only requires two simple clicks, select the menu option, and hit the `Update Pipeline` button.
+
+:::info
+Keep in mind This will not affect your existing setup, a new bundle will be created for the new release.
```
Comment (Innders) [inline website/docs/admin_server_updating_pipeline.md:16]: I feel there is a better word than "setup". Maybe "current production configuration"?.

Also small typo on "This" needs lowercase "this".

File: website/docs/admin_server_updating_pipeline.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+The `Update Pipeline` UI is pre-configured, but you can adjust it by clicking the `change` button next to `Addons` and `AYON launchers`.
+
+![](assets/server/admin/update_pipeline_configuration.png)
+
+:::info
+Once the update begins, it should not be interrupted. In the rare event the update fails to complete, restarting the server will cancel the download process.
+:::
+
+:::tip Next Steps
+Once the update is complete, you can begin testing it by setting it to staging. A pop-up window will appear to help you copy settings from existing bundles. The same process applies when setting it to production, but be aware that this may affect your current setup.
```
Comment (Innders) [inline website/docs/admin_server_updating_pipeline.md:30]: Maybe just a slight rewording :)

"After the update is complete, you can start testing by setting the bundle to staging. A dialog will appear, allowing you to copy settings from existing bundles. Once you’re satisfied with the changes, set the bundle to production using the same process."

ChatGPT is your friend ;)

**Review by Innders — CHANGES_REQUESTED** [review]
Great stuff, just some tiny changes to make it perfect 👌

File: website/docs/admin_server_updating_pipeline.md (hunk 4 above)
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md]: Oh thank you. yeah, "current production configuration" sounds better. 
I've added it in https://github.com/ynput/ayon-documentation/pull/266/commits/53e99d4b6d0a9c59e8a9b736dd73176f15172978

File: website/docs/admin_server_updating_pipeline.md (hunk 5 above)
Comment (MustafaJafar) [reply website/docs/admin_server_updating_pipeline.md]: Thank you https://github.com/ynput/ayon-documentation/pull/266/commits/53e99d4b6d0a9c59e8a9b736dd73176f15172978



> ChatGPT is your friend ;)

Poor Clickup's ai assistant will have no friends then 😅

**Review by Innders — APPROVED** [review]

**Pull request merged**